migrator/migrations/n_09_to_n_10_postgres_cluster_cves/migration.go

```diff
@@ -0,0 +1,136 @@
+// Code originally generated by pg-bindings generator.
+
+package n9ton10
+
+import (
+	"context"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	cveUtil "github.com/stackrox/rox/migrator/migrations/cvehelper"
+	"github.com/stackrox/rox/migrator/migrations/loghelper"
+	legacy "github.com/stackrox/rox/migrator/migrations/n_09_to_n_10_postgres_cluster_cves/legacy"
+	pgStore "github.com/stackrox/rox/migrator/migrations/n_09_to_n_10_postgres_cluster_cves/postgres"
+	"github.com/stackrox/rox/migrator/types"
+	rawDackbox "github.com/stackrox/rox/pkg/dackbox/raw"
+	pkgMigrations "github.com/stackrox/rox/pkg/migrations"
+	pkgSchema "github.com/stackrox/rox/pkg/postgres/schema"
+	"github.com/stackrox/rox/pkg/sac"
+	"gorm.io/gorm"
+)
+
+var (
+	migration = types.Migration{
+		StartingSeqNum: pkgMigrations.CurrentDBVersionSeqNumWithoutPostgres() + 9,
+		VersionAfter:   storage.Version{SeqNum: int32(pkgMigrations.CurrentDBVersionSeqNumWithoutPostgres()) + 10},
```

Comment:
same as earlier comment. It would be helpful if you add a comment about why  `+9` and `+10`